catalyst/core/callbacks/metrics.py

```diff
@@ -68,27 +78,62 @@ def __init__(
     @property
     @abstractmethod
     def metric_fn(self):
-        """@TODO: Docs. Contribution is welcome."""
+        """
+        Specifies used metric function.
+        """
         pass
```

Comment:
maybe it is better to use `raise NotImplementedError`?

Reply:
it's `abstractmethod`, so there is no opportunity to create this class without `metric_fn` reimplementation